### ISSUE TITLE
Wings 3d jscad exporter quad polygons polyhedron

### ISSUE THREAD

**Issue by gilboonet** [jscad/OpenJSCAD.org#522]
Wings 3d jscad exporter produces polyhedron having quad polygons, they are correctly rendered, but make jscad hang when used on a csg operation. So I wrote a little function to change quad polygons from a polyhedron to triangular and the csg operation work.

## Expected Behavior
It can be tested here : https://openjscad.org/#https://raw.githubusercontent.com/gilboonet/gilboonet.github.io/master/demos/test_test_wings.jscad where I changed the polyhedron function call to polyhedron_tri

## Actual Behavior
into the same script, if you replace polyhedron_tri by polyhedron (as it is when freshly exported), jscad freezes.

## Steps to Reproduce the Problem

  1. make a design with Wings 3d
  2. export it to .jscad
  3. import the .jscad into OpenJSCAD.org
  4. replace main() by volume()
  5. make a new main() using volume on a csg operation (I used intersection)
## Specifications

  - Version: latest V1
  - Platform: Linux Ubuntu 19.10
  - Environment: Firefox 73

**Comment by z3dev** [comment]
@gilboonet thanks. This is interesting but I have a few questions.

There’s an internal toTriangles() function for V1 CSG objects. Have you tired to use that?

We are considering a similar function for V2. Would this function be useful?

**Comment by gilboonet** [comment]
As it was straightforward to do directly from the exported data (only tri or quad polygons) before the csg was built, I didn't try to to it with csg functions. I only first tried to retesselate the polyhedron but didn't find the proper syntax. I'm not sure it would be useful, it's the first time I need to triangulate polygons, whenever possible I use triangular polygons in order to keep things simple. I really need to start to use V2, is there something to install to use jscad.xyz ? (I tried but it didn't find files corresponding to the requires, anyway I can wait that a first public version become available)
